ran-core/src/main/java/io/ran/Clazz.java

```diff
@@ -413,31 +527,40 @@ public Type getOwnerType() {
 		};
 	}
 
+	/**
+	 * @param ofClazz superclass or generic interface of `this`
+	 * @return Clazz of `ofClazz` with generics set as specified by `this`
+	 */
 	Clazz<?> findGenericSuper(Class<?> ofClazz) {
+		if (ofClazz.equals(Object.class)) { // todo I added this, is it an issue?
```

Comment:
@todo: This should be safe to add, as far as I can tell

Reply:
:+1: